### DISCUSSION TITLE
Make an android app with this

### DISCUSSION THREAD

**Discussion by thibaultmol** [timschneeb/GalaxyBudsClient#437] [General]
I KNOW it sounds stupid. But hear me out:
The official android app works well EXCEPT for the fact that it requires A TON of permissions... 
Permissions that it shouldn't need, like 'sms messages'.. I don't want it to read out sms messages, nor do I have it enabled. But the app refuses to work if you don't give it that permission (even if you disable it afterwards).

i'm not the only person complaining about this, the app itself has a 2.x rating because many people complain about the usage of permissions that it doesn't need:
![image](https://github.com/ThePBone/GalaxyBudsClient/assets/1010226/b6b02124-566b-41d7-8309-14592def6db4)
![Screenshot 2023-12-14 at 19-15-31 Galaxy Buds2 Manager – Apps on Google Play](https://github.com/ThePBone/GalaxyBudsClient/assets/1010226/b8229d58-75fc-4889-8e1b-48ac05477f9f)
![Screenshot 2023-12-14 at 19-15-47 Galaxy Buds2 Manager – Apps on Google Play](https://github.com/ThePBone/GalaxyBudsClient/assets/1010226/c1f3406b-6562-49da-8ad5-f537631de03e)
![Screenshot 2023-12-14 at 19-15-58 Galaxy Buds2 Manager – Apps on Google Play](https://github.com/ThePBone/GalaxyBudsClient/assets/1010226/954ef8d9-d1c1-4b67-9615-fef315c0de43)
![Screenshot 2023-12-14 at 19-16-05 Galaxy Buds2 Manager – Apps on Google Play](https://github.com/ThePBone/GalaxyBudsClient/assets/1010226/390b2403-6fe4-4a82-9e1a-7fed557d67be)

**Comment by OverStyleFR** [comment]
I agree with this! Make an android app! Your software show and provide more information than the official one!
